src/_static/docker-compose-22.4.yml

```
@@ -1,65 +1,65 @@
 services:
   vulnerability-tests:
-    image: greenbone/vulnerability-tests
+    image: registry.community.greenbone.net/community/vulnerability-tests:stable
```

Please just leave the tags as they where ... this means in this case: No tag.

```suggestion
    image: registry.community.greenbone.net/community/vulnerability-tests
```